components/DailyCrusher/DailyCrusher.js

```diff
@@ -0,0 +1,171 @@
+      <StyledArticleWrap>
+        <StyledDescriptionText>
+          Get in shape with our crusher workouts! Every day we suggest a
+          different workout for you to try out.
+        </StyledDescriptionText>
+
+        <StyledLink href="/workoutlist">
+          <Workout workout={dailyCrusher} spotlightMode={true} />
+        </StyledLink>
+        <StyledPostionRelative>
+          {isCrusherDetailsVisible ? null : (
+            <StyledAbsoluteToggleButton onClick={toggleVisibility}>
+              Show More
+            </StyledAbsoluteToggleButton>
+          )}
+          <StyledPreviewFilter
+            $showMore={
+              isCrusherDetailsVisible ? "filter: none; color: black;" : null
+            }
+          >
+            <h3>{dailyCrusher.title}</h3>
+            <StyledDescriptionText>
+              {dailyCrusher.description1}
+            </StyledDescriptionText>
+          </StyledPreviewFilter>
+        </StyledPostionRelative>
+        {isCrusherDetailsVisible ? (
+          <>
+            {dailyCrusher.exercises.map((exercise) => {
+              return (
+                <StyledExerciseSetsReps key={uid()}>
+                  <StyledBoldText>{exercise.name}: </StyledBoldText> Sets:{" "}
+                  {exercise.sets} / Reps: {exercise.reps}
+                </StyledExerciseSetsReps>
+              );
+            })}
+            <StyledDescriptionText>
+              {dailyCrusher.description2}
+            </StyledDescriptionText>
+            <StyledDescriptionText>
+              {dailyCrusher.description3}
+            </StyledDescriptionText>
+          </>
+        ) : null}
+      </StyledArticleWrap>
+      {!isCrusherDetailsVisible ? null : (
+        <StyledToggleButton onClick={toggleVisibility}>
+          Show Less
+        </StyledToggleButton>
+      )}
+    </StyledCrusherWrapper>
+  );
+}
+
+const StyledCrusherWrapper = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 12px;
+  max-width: 600px;
+  margin: 20px auto;
+  text-align: center;
+  box-shadow:
+    0 4px 8px #0000001a,
+    0 -0.5px 5px #0000000d;
+`;
+
+const StyledArticleWrap = styled.div`
+  padding: 0 2rem;
+`;
+
+const StyledDescriptionText = styled.p`
+  text-align: left;
+  padding-bottom: 0.5rem;
+`;
+
+const StyledExerciseSetsReps = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: #f4f4f4;
+  border-radius: 4px;
+  margin-bottom: 0.5rem;
+  padding: 0.5rem;
+  align-items: center;
+`;
+
+const StyledBoldText = styled.span`
+  font-weight: bold;
+  padding-right: 0.5rem;
+`;
+
+const StyledPreviewFilter = styled.div`
+  filter: blur(1px) brightness(1.2);
+  color: grey;
+  ${(props) => props.$showMore}
+`;
+
+const StyledPostionRelative = styled.div`
+  position: relative;
+`;
+
+const StyledAbsoluteToggleButton = styled(ToggleButton)`
+  position: absolute;
+  top: 20px;
+  left: 50%;
```

Comment:
For centering horizontally I like to use `margin: 0 auto` or `left: 0; right: 0;`

Reply:
This does not work here since we are working with position: absolute, meaning here it is necessary to work with top/left etc. If we were to use left: 0 & right: 0, the button would span across the div, which is not the look we are looking for :)